docs/src/app/components/pages/discover-more/Showcase.jsx

```diff
@@ -42,6 +42,12 @@ const appList = [
     img: 'images/showcase/cloudcraft.png',
     link: 'https://cloudcraft.co/',
   },
+  {
+    title: 'It\'s quiz',
+    author: 'It\'s quiz',
+    img: 'images/showcase/itsquiz.png',
+    link: 'https://app.itsquiz.com/',
```

Comment:
Could you link to your home-page (http://itsquiz.com), also your site doesn't support SSL. (You probably should, google are going to start penalising sites that don't! :)